src/components/SequenceCard/SequenceCard.js

```diff
@@ -52,8 +64,13 @@ const Card = styled.section`
   border: 1px solid grey;
   margin: 15px 0;
   padding: 0 10px;
+  ${(props) => !props.isActive && 'background-color : #E7C5CA; color: grey;'}
 `
 
+const Header = styled.header`
+  display: flex;
+  justify-content: space-between;
+`
 const Timecode = styled.p`
   font-weight: 800;
```

Comment:
Könntest du bei der Font nicht em / rem arbeiten und die über die global styles festlegen? (Evtl liegt deine Prio da auch gar nicht drauf, in dem Fall gerne ignorieren :) )

Reply:
Yes! You're not the first, who comment on this one :-D i will do that when the design is ready. right now the functionality of my code is more important for me me than the design ;-)